### PR TITLE
Remove useless return value of gc task function

### PR DESCRIPTION
**Pull request by SataQiu**
Signed-off-by: SataQiu <qiushida@beyondcent.com>

### Ⅰ. Describe what this PR did
Remove useless return value of gc task function

### Ⅱ. Does this pull request fix one issue?

#845

### CODE REVIEW

File: supernode/daemon/mgr/gc/gc_task.go [hunk 1]
```diff
@@ -49,17 +49,15 @@ func (gcm *Manager) gcTasks(ctx context.Context) {
...
 		removedTaskCount++
 	}
 
 	logrus.Infof("gc tasks: success to full gc task count(%d), remainder count(%d)", removedTaskCount, totalTaskNums-removedTaskCount)
 }
 
-func (gcm *Manager) gcTask(ctx context.Context, taskID string) bool {
-	logrus.Infof("start to gc task: %s", taskID)
+func (gcm *Manager) gcTask(ctx context.Context, taskID string) {
+	logrus.Infof("start gc task: %s", taskID)
```
Comment (Starnop) [inline supernode/daemon/mgr/gc/gc_task.go]: IMOP,  the correct gramma should be `start to do something`.

Comment (yunfeiyanggzq) [reply supernode/daemon/mgr/gc/gc_task.go]: `
logrus.Infof("gc tasks: success to full gc task count(%d), remainder count(%d)", removedTaskCount, totalTaskNums-removedTaskCount)
`
I think add a info-prefix may be better like this,and logrus.Debug() may be better

**Comment by Starnop** [comment]
@SataQiu Could you please rebase code and update it. And then `push --force` again.

**Comment by SataQiu** [comment]
Done :) @Starnop

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**